hyperion-geiger-counter/src/main/java/com/willowtreeapps/hyperion/geigercounter/GeigerCounterModule.java

```diff
@@ -0,0 +1,70 @@
+package com.willowtreeapps.hyperion.geigercounter;
+
+import android.os.Build;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Toast;
+
+import com.willowtreeapps.hyperion.plugin.v1.PluginModule;
+
+class GeigerCounterModule extends PluginModule implements View.OnClickListener {
+
+    private View view;
+    private DroppedFrameObserver observer;
+
+    // Helpers
+
+    private static boolean arePrerequisitesAvailable() {
+        return DroppedFrameObserver.TARGET_API <= Build.VERSION.SDK_INT;
```

Comment:
@Kritarie maybe this warrants a more generic implementation for plugins to describe their minimum supported API level. A default implementation can easily match the library minimum and permit an opt into higher levels such as this plugin requires.

Reply:
Yeah, I've been thinking about this but not sure what the best implementation is. Was thinking `hyperion-core` could check a min SDK annotation on the `Plugin` before invoking anything on it. Or potentially the `Plugin` just has some `getMinSdk` method the dev can override.

Reply:
That kind of precedent would have simplified things for me. 👍